site/pages/about-us/timeline-slice/navigator/item/style.css

```diff
@@ -11,9 +11,11 @@
   transition: 0.3s ease-in-out;
   left: -15px;
   cursor: pointer;
+  width: 36px;
+  height: 50px;
 }
```

Comment:
@dstodolny Clever! but I think the height is too big - compare it to https://red-badger.com/about-us